python/scripts/rfi_toggle_cron.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+import requests
+import json
+import time
+
+#Wait until the correct UTC time (deals with daylight savings time)
+while(datetime.datetime.utcnow().time() < datetime.time(19,10)):
```

Comment:
If only there was a way of calculating and sleeping for the exact number of seconds between now and 19:10....